index.js

```diff
@@ -201,3 +201,46 @@ modal.addEventListener('click', (e) => {
 });
 
 generateProduct();
+
+const button = document.querySelector('#form_button');
+
+button.addEventListener('click', (event) => {
+  event.preventDefault();
+  // form validation
+  const contactForm = document.getElementById('contactForm');
+  const fullName = document.querySelector('#name');
+  const email = document.querySelector('#email');
+  const textarea = document.querySelector('#textarea');
+  const messageErr = document.querySelector('#message');
+  const checkLower = email.value;
+
+  if (fullName.value === '') {
+    messageErr.innerHTML = 'Please provide your name';
+    fullName.focus();
+    // return false;
+  } else if (email.value === '') {
+    messageErr.innerHTML = 'Please provide email';
+    email.focus();
+    // return false;
+  } else if (checkLower !== checkLower.toLowerCase()) {
+    messageErr.innerHTML = 'Please use lowercase for email';
+    email.focus();
+    // return false;
+  } else if (!email.value.includes('@') || !email.value.includes('.')) {
+    messageErr.innerHTML = 'Enter a valid email in Lowercase';
+    messageErr.classList.add('show');
+    email.focus();
+    // return false;
```

Comment:
- [ ] Kindly remove all the return commented code lines, this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻